src/libstd/keyword_docs.rs

```diff
@@ -360,7 +383,7 @@ mod for_keyword { }
 
 #[doc(keyword = "if")]
 //
-/// If statements and expressions.
+/// Traditional control flow based on boolean conditions.
```

Comment:
Stops being true with https://github.com/rust-lang/rust/issues/53667 and is already false with `if let`.

Reply:
True, this should probably just stay the way it is. I was trying to differentiate it from `match`.

Reply:
It's a good idea to differentiate, but perhaps say:
```suggestion
/// Evaluate a block if the condition holds or possibly a different one otherwise.
```